src/libmongoc/tests/unified/operation.c

```diff
@@ -2518,6 +2526,19 @@ operation_loop (test_t *test,
    return false;
 }
 
+static bool
+operation_assert_number_connections_checked_out (test_t *test,
+                                                 operation_t *op,
+                                                 result_t *result,
+                                                 bson_error_t *error)
+{
+   /* "This operation only applies to drivers that implement connection pooling
+    * and should be skipped for drivers that do not."
+    * TODO: (CDRIVER-3525) add this assertion when CMAP is implemented. */
+   result_from_ok (result);
+   return true;
```

Comment:
If the tests should be skipped, should this just raise an error?

Reply:
We can still run the test and make other assertions. I have a preference for keeping this as-is.